### PR TITLE
superficial english fixes

### PR DESCRIPTION
**Pull request by jslmann**
because I appreciate your work, and was bored..

### CODE REVIEW

File: docs/source/examples/Notebook/JavaScript Notebook Extensions.rst [hunk 1]
```diff
@@ -1,38 +1,37 @@
...
-One of the main reason that allowed us to developp the current notebook
-web application was to embrase the web technology.
+One of the main reasons that we developed the current notebook
+web application was to embrace  web technology.
 
-By beeing a pure web application using HTML, Javascript and CSS, the
-Notebook can get all the web technology improvement for free. Thus, as
-browsers support for different media extend, The notebook web app should
+By being a pure web application using HTML, Javascript and CSS, the
+Notebook can usell the web technology improvement for free. Given
```
Comment (rgbkrk) [inline docs/source/examples/Notebook/JavaScript Notebook Extensions.rst]: s/usell/use

**Comment by Carreau** [comment]
Thanks, Ill merge that manually later as we have other fixes to same file in progress !

**Comment by rgbkrk** [comment]
Awesome thank you!

**Comment by Carreau** [comment]
Thanks! cherry-picked and manually merged as part of #363 !

Closing !

**Pull request closed without merging**